Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -1077,7 +1077,10 @@ def get_script_code(self, script_uid) -> Dict[str, Any]:
 
         return reply.get('reply')
 
-    def run_script(self, script_uid, endpoint_ids: list, timeout: int, parameters: Dict[str, Any]) -> Dict[str, Any]:
+    @logger
+    def run_script(self,
```

Comment:
why we had this function if it wasn't used?

Reply:
was implemented before api was ready